pinot-common/src/test/java/com/linkedin/pinot/common/utils/FileUploadUtilsTest.java

```diff
@@ -15,79 +15,66 @@
  */
 package com.linkedin.pinot.common.utils;
 
+import com.linkedin.pinot.common.utils.FileUploadUtils.FileUploadType;
+import com.sun.net.httpserver.Headers;
```

Comment:
com.sun?

Reply:
They are used to mock a http server (not changed in this PR).